tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/activities.py

```diff
@@ -28,6 +29,14 @@ def __init__(self, marionette):
         # TODO Difficult intermittent bug 977052
         time.sleep(1)
 
+    def tap_report(self):
+        #Wait(self.marionette).until(
```

Comment:
Commented code. We usually don't commit code that is commented out. Otherwise, the next reader will probably ask:
* Is this code important?
* Will it come back again?
* Should I have to uncomment it?

We can make his reading more comfortable by removing this code